### DISCUSSION TITLE
in the config.json

### DISCUSSION THREAD

**Discussion by ghost** [darylhjd/mangadesk#62] [Q&A]
Is it just me, or does setting explicitContent to true not automatically set show explicit tick when in search?

**Comment by darylhjd** [comment]
actually this is the intended behaviour now (refer to CONFIG.md in app/core folder).

the idea was for the search to only follow that search-specific setting. (for example, you may have explicitContent set to true but you didnt want to search for explicit content)

in fact, explicitContent only affects listings in the main page for a guest (actually its not very useful imo, which was probably the main reason i forgot to document it in the first place in earlier versions of the app; i probably intended to remove it).

but now that you mention it, i could set the default explicit content option in search to true if the user has set that setting to true.

**Reply by ghost** [reply]
I just found it a bit odd, but the app is incredible! Thank you for providing the incredible app.

**Reply by darylhjd** [reply]
the option used to be used to load explicit-tagged chapters as well (not just explicit-tagged manga) but that is no longer to case (it now uses the content rating of the manga to load explicit/non-explicit chapters).

so yea, just a legacy thing i guess haha
